.github/workflows/config/release.yaml

```diff
@@ -1,7 +1,7 @@
 release:
   branching:
     execution:
-      time: "01:00:00"
+      time: "17:00:00"
     schedule:
       - on: "2023-01-28"
```

Comment:
```suggestion
      - on: "2023-01-27"
```

Reply:
11 AM CST is 5 PM UTC right @nathanklick  ? Why do we need to change date 🤔